### ISSUE TITLE
Estado virtual

### ISSUE THREAD

**Issue by guilhermezm** [ancap-devs/forum#3]
Ola ancaps! Surgiu uma idea de um estado (país) em que todos os cidadãos e empresas são virtuais, ele teria sua própria moeda virtual descentralizada (como Bitcoin), e tudo funcionária baseado na blockchain com servidores descentralizados e a aprovação de novos cidadãos e empresas seriam feitas por votações de nossos já residentes. Seria cobrado uma parte do faturamento para pagar quem aloca seus servidores para que o país fique no ar, Também seria opensurce e sem nenhum grupo 100% no comando. É um projeto grande e complicado, mas acredito que com um grupo grande e com pessoas boas conseguiríamos dar vida a “CyberLand”. Também sei que esse projeto não bate completamente com as ideias do Ancap, mas pelo menos já seria um gigantesco passo para a liberdade! Impostos muito muito baixos, sem burocracia, muito rapidez, sigilo e segurança (coisas que quase nenhum país oferece hoje!). O que acham?

**Comment by BrOrlandi** [comment]
Legal a ideia mas tem uns pontos que chamam atenção:

- Os residentes teriam que aprovar entrada de novas pessoas ou empresas pq? Como se todos fossem donos de tudo, me passou uma ideia meio comunista até.
- Cobrar uma taxa das empresas com base no faturamento, é puramente imposto.
- Pq realmente precisaria ter impostos?
- Como vc disse, não batem totalmente com ideias anarcocapitalista, na verdade em quase nada, seria só um novo Estado no limite até meio comunista por querer regular a entrada de pessoas e empresas.

Sugestões pra melhorar sua ideia, tira essa ideia de controle, se o negócio for bom isso vai atrair pessoas e empresas, não necessariamente vc precisaria barrar elas, pq vc barraria? A ideia do capitalismo é excluir pessoas e empresas justamente não comprando delas ou prestando serviço a elas, vc não precisa de uma força de estado pra isso.
Tira essa de taxar faturamento e impostos, isso somado a ideia q todos serão donos é bem comunista.
O que é mais interessante é uma rede de negócios totalmente feita por contratos em blockchain e negocia por uma criptomoeada, isso garante mais liberdade, menos burocracia, rapidez, sigilo e segurança como você propôs. Mas acredito que já existam coisas nessa linha, recomendo pesquisar mais sobre e se encontrar algo, por favor, compartilha nessa thread pra evoluirmos a ideia.

**Comment by guilhermezm** [comment]
> Legal a ideia mas tem uns pontos que chamam atenção:
> 
> 
> 
> - Os residentes teriam que aprovar entrada de novas pessoas ou empresas pq? Como se todos fossem donos de tudo, me passou uma ideia meio comunista até.
> 
> - Cobrar uma taxa das empresas com base no faturamento, é puramente imposto.
> 
> - Pq realmente precisaria ter impostos?
> 
> - Como vc disse, não batem totalmente com ideias anarcocapitalista, na verdade em quase nada, seria só um novo Estado no limite até meio comunista por querer regular a entrada de pessoas e empresas.
> 
> 
> 
> Sugestões pra melhorar sua ideia, tira essa ideia de controle, se o negócio for bom isso vai atrair pessoas e empresas, não necessariamente vc precisaria barrar elas, pq vc barraria? A ideia do capitalismo é excluir pessoas e empresas justamente não comprando delas ou prestando serviço a elas, vc não precisa de uma força de estado pra isso.
> 
> Tira essa de taxar faturamento e impostos, isso somado a ideia q todos serão donos é bem comunista.
> 
> O que é mais interessante é uma rede de negócios totalmente feita por contratos em blockchain e negocia por uma criptomoeada, isso garante mais liberdade, menos burocracia, rapidez, sigilo e segurança como você propôs. Mas acredito que já existam coisas nessa linha, recomendo pesquisar mais sobre e se encontrar algo, por favor, compartilha nessa thread pra evoluirmos a ideia.

Verdade! Eu dei uma pesquisada e não achei nada do tipo, te interessa fazer este projeto com essas melhorias? Eu sei front-end, estou aprendendo php agr, então não sei se teria muito a agregar, mas qualquer coisa estarei totalmente disposto a ajudar!
